### DISCUSSION TITLE
Zonas interesantes para 1º procesado

### DISCUSSION THREAD

**Discussion by manudona92** [manudona92/MapantES#26] [General]
Como hablamos ayer, a parte de Canarias entera, en la parte peninsular habia pensado en estos mapas de orientación: 

- Navaleno y S. Leonardo de Yague(Soria) http://kartarkiv.nydalen.idrett.no/show_map.php?user=Ragne+Wiklund&map=2658

- Urbasa (Navarra) http://www.liganorteorientacion.org/doma/show_map.php?user=ivancobi&map=593

- San Rafael (Segovia) http://olle.server.ee/files/kaart000600.jpg

- Guils de Cerdanya ( Girona) http://web.entrebalizas.org/Doma/show_map.php?user=llaveriamagi&map=2040

- Barbate (Cádiz) http://karten.guedels.ch/show_map.php?user=T%C3%A9rence&map=960

- Pinares Valladolid http://omaps.worldofo.com/?id=76335

- Coto Maravillas (Murcia)

La semana que viene os dejaré que PULLAUTA.INI utilizar, qué cuadrantes coger y de que fecha. Si tenéis algun otro terreno interesante, bienvenido es. El tamaño serán unos 4 x4 tiles aprox.

**Comment by jarufe** [comment]
La prueba media del CEMTBO 2020 se hizo en parte de un mapa ya existente que publicaron en la web del campeonato. Tiene muchos cultivos y además catalogados con distintos símbolos que los delimitan (cultivo, frutales, viñedos). Podría ser interesante para probar lo del SIGPAC. [https://trofeoquijotesmtbo.files.wordpress.com/2020/09/mapa-arroyo-de-la-canada.pdf](https://trofeoquijotesmtbo.files.wordpress.com/2020/09/mapa-arroyo-de-la-canada.pdf)

**Reply by jarufe** [reply]
La localidad más al SE es Casas de Ves. Coordenadas 39.257870;-1.333064

**Reply by agustincaballero** [reply]
Ok, lo pondré, es interesante porque los mapas de MTBO van a salir bastante parecidos

**Comment by agustincaballero** [comment]
Si podéis, cuando pongáis por aquí una zona adjuntad un fichero kml, o geojson, etc. con un polígono que delimite la zona en cuestión

**Reply by manudona92** [reply]
Me he crecido un poco y he añadido Lagos de Covadonga y Bárdenas Reales 

Github no acepta KML, asi que te lo dejo por wetransfer: https://we.tl/t-l2WafQRo1I

Respecto los pullauta .INI, Los estoy dejando en la portada del repositorio. Galicia y Canarias deben de estar ya con 3 verdes, tengo que revisar el resto. Mañana yo creo que te puedo dejar ya los pullauta de las zonas y de que cobertura puedes descargar los datos (De Navarra, 100% seguro, descargate los de la 1 Cobertura, 2012)

**Reply by agustincaballero** [reply]
Manu, preferiría un archivo kml por cada zona, que tenga como nombre del archivo el nombre que le demos al "mapa".
Se puede subir aquí renombrándolo a txt.
Es que si no es un lío para mí.

**Comment by manudona92** [comment]
He cambiado Calahorra (La Rioja) por Bárdenas Reales. Estan todas las zonas individualizadas en su KML en este enlace de WeTransfer para descargar: https://we.tl/t-82pFeLHSyn

Respecto coberturas (Zona / Cobertura / Pullauta a utilizar) : 

- **Lagos de Covadonga**
- **Urbasa** 1º Cobertura 2012 / Navarra_2.ini
- **Pinares Valladolid** 2º Cobertura  (Intentar cargar los LAZ reclasificados de IGN) /Madrid_1.ini
- **Calahorra** 2º obertura 2016  (Intentar cargar los LAZ reclasificados de IGN) / Rioja.ini
- **Guils de Cerdanya** 2º Cobertura 2015 / Madrid_1_1.ini
- **Navaleno** 1º Cobertura 2010 / Madrid_1_1.ini
- **BMT Albacete**
- **Barbate** 1º Cobertura 2015 / Murcia_1.ini
- **Peguerinos**
- **Coto Maravillas** 2º Cobertura 2016 (Intentar cargar los LAZ reclasificados de IGN)/ Murcia_1.ini

Los que digo intentar cargar LAZ refinados, es que cuando lo publiquemos, procesemos si o si (porque los publicados tienen mucho ruido) con los datos reclasificados por el IGN, y que tenemos que probar con estas pruebas cuando hagamos las pruebas de carga. 

Los que faltan te los paso durante la semana, pero con los otros ya puedes empezar a preparar las zonas.
Todos los pullauta están actualizados aquí: 
![image](https://user-images.githubusercontent.com/76484741/104960489-97d21b00-59d4-11eb-9e54-e2a534faa7b8.png)

**Reply by manudona92** [reply]
Respecto las que me quedaban: 

-**Peguerinos** 2º Cobertura 2017 (Intentar cargar los LAZ reclasificados de IGN) /Madrid_1.ini
- **Lagos Covadonga**  1º Cobertura 2012 / Asturias_1.ini
- **BMT Albacete CEO**  1º Cobertura 2012 / Madrid_1.ini

De los tiles que no podamos adquirir los datos LAZ reclasificados, **mejor utilizar la antigua cobertura** ya que los de la 2º sin reclasificar (no hemos probado los reclasificados aún) tiene excesiva cantidad de ruido.

**Comment by agustincaballero** [comment]
Cuando dices los LAZ reclasificados, te refieres a que nos tienen que mandar un shapefile nuevo con la ubicación de los LAZ nuevos, ¿no?

**Comment by manudona92** [comment]
Como comentábamos ayer, te dejo el KML con las zonas test para probar la 2º Cobertura de los siguientes lotes. Todas de ellas tienen que estar disponibles en CDG (algunas son reclasificadas y otras clasificación por defecto)

- MAD
- CAT
- GAL E, W
- ARA C, NC, SE
- CYL SE
- EXT S
- RIOJA
- CV ALAC, VLC, CAST
- MUR
- BALEARES

KML: https://we.tl/t-DeAyhstbxV

Viene dividio los KML por lotes / carpeta

Utiliza el INI de Madrid para casi todos excepto GAL E, W (Galiciaxx.ini) y MUR (reprocesa con Coto) y CV Alicante que puedes utilizar el de Murcia XX ini. He añadido una zona más a CYL SE a parte de Pinares y Peguerions que ya estaba. 

Los de País Vasco y CLM hay que pedirlo al IGN que nos lo vuelquen, miraré que tiles coger. Depende como vaya este test de ruido, pediremos que nos vuelquen más que tienen clasificadas en su servidor pero que aun no estan en CDG.

**Reply by manudona92** [reply]
Según el último cuadro de orden de procesamiento y de qué cobertura (1º, 2º) necesitamos hacer unos últimos test: 

Determinar si es válido Madrid_2.ini para los **siguientes lotes de la 1º Cobertura**: 

- Murcia (reprocesar Coto Marivallas o zonas de sondeo, anteriomente procesadas con Murcia_1.ini)
- Comunidad Valenciana (reprocesar zonas de sondeo, anteriomente reprocesadas con 2º cobertura)
- ARA C (reprocesar zona de sonde-Calatayud- pero con 1º cobertura)
- ARA SE (reprocesar zona de sondeo-Matarraña-pero con 1º cobertura)
- Andalucía (reprocesar Barbate pero con Madrid_2.ini, anteriormente procesada con Murcia_1.ini)
- Extremadura (reprocesar zonas de sondeo pero con 1º cobertura)
- Madrid (reprocesar zonas de sondeo pero con 1º cobertura)

Todas las anteriores procesar con Madrid_2.ini

Por otro lado:
- Rioja (reprocesar zona de sondeo -Lumbreras- con parámetro Rioja_1.ini)
